src/main/java/persistence/Application.java

```diff
@@ -28,20 +30,21 @@ public static void main(String[] args) {
             TableScanner tableScanner = persistenceConfig.tableScanner();
             Set<EntityNode<?>> nodes = tableScanner.scan(BASE_PACKAGE);
 
-            QueryBuilder queryBuilder = persistenceConfig.queryBuilder();
+            QueryBuilderFactory factory = QueryBuilderFactory.getInstance();
             for (EntityNode<?> node : nodes) {
-                String createTableQuery = queryBuilder.buildCreateTableQuery(node);
+                String createTableQuery = factory.buildQuery(QueryType.CREATE, new SimpleMetadataLoader<>(node.entityClass()), null);
```

Comment:
```suggestion
                String createTableQuery = factory.buildQuery(QueryType.CREATE, new SimpleMetadataLoader<>(node.entityClass()));
```
리팩토링된후 코드 점검도 필요해보여요~👀